pulpcore/cli/rpm/repository.py

```diff
@@ -126,6 +126,7 @@ def repository(ctx: click.Context, pulp_ctx: PulpContext, repo_type: str) -> Non
     click.option("--description"),
     click.option("--retain-package-versions", type=int),
     remote_option,
+    click.option("--metadata-signing-service", type=str),
```

Comment:
Can you possibly make this a `resource_option` like the `remote_option`? This would allow users to supply either the name or the href of the signing service. You could use this context: https://github.com/pulp/pulp-cli/blob/main/pulpcore/cli/core/context.py#L314 for the `context_table`.

Reply:
This option probably needs a version guard (`needs_plugins=PluginRequirement("rpm", min=???)`).

cc @pulp/rpm Can you verify this?

Reply:
> This option probably needs a version guard (`needs_plugins=PluginRequirement("rpm", min=???)`).
> 
> cc @pulp/rpm Can you verify this?

metadata_signing_service has been in pulp_rpm since forever (def pre-3.9) - so we're good.